cpp/src/slice2cs/Gen.cpp

```diff
@@ -1589,7 +1695,10 @@ Slice::Gen::TypesVisitor::visitInterfaceDefStart(const InterfaceDefPtr& p)
     string ns = getNamespace(p);
 
     _out << sp;
-    writeDocComment(p);
+    ostringstream notes;
+    notes << "Use the methods of this interface to invoke operations on a remote Ice object that implements <c>"
```

Comment:
What about using `<see cref=...` here instead of `<c>`

Reply:
`p->name();` is the Slice interface name. I don't want to reference the server-side C# interface from the proxy.